Lib/test/test_fractions.py

```diff
@@ -155,11 +164,13 @@ def testInit(self):
         self.assertRaises(TypeError, F, "3/2", 3)
         self.assertRaises(TypeError, F, 3, 0j)
         self.assertRaises(TypeError, F, 3, 1j)
+        self.assertRaises(TypeError, F, 1j, 3)
```

Comment:
Can you add:
- tests to init from a single string
- tests to init from two arguments when one/both are instances of: Fraction, int, Decimal

Reply:
Such tests already exist in other parts of `Lib/test/test_fractions.py`

Reply:
If there is a specific test that you think is missing, let me know and I'll add it.

Reply:
Yeah, you're right, I missed that -- thanks!